src/thx/Arrays.hx

```diff
@@ -1160,6 +1161,17 @@ Finds the min element of the array given the specified ordering.
 
     return Options.toFailure(Nel.fromArray(collisions), m);
   }
+
+  public static function toStringMap<V>(arr: ReadonlyArray<Tuple<String, V>>): StringMap<V> {
```

Comment:
`StringMap` is the underlying type, but I am not sure it is reconciled transparently. If you are using it in a place where `Map<String, T>` then I am fine with it.